.install-hooks.sh

```
@@ -0,0 +1,16 @@
+#!/bin/sh
+REPO_DIR=$(git rev-parse --show-toplevel)
+GIT_DIR=$REPO_DIR/.git
+VENV_ACTIVATE=$VIRTUAL_ENV/bin/activate
+if [[ ! -f $VENV_ACTIVATE ]]
```

Shellcheck (0.11.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)